apollo-federation/tests/composition/compose_misc.rs

```diff
@@ -418,3 +419,51 @@ fn composes_subgraphs_with_directives_on_renamed_root_types() {
     let result = compose_as_fed2_subgraphs(&[subgraph_a, subgraph_b]);
     let _supergraph = result.expect("Expected composition to succeed");
 }
+
+#[test]
+fn composes_subgraph_with_explicit_tag_spec_link() {
+    let subgraph_with_tag_link = r#"
+        extend schema
+            @link(url: "https://specs.apollo.dev/link/v1.0")
+            @link(url: "https://specs.apollo.dev/tag/v0.2")
+
+        directive @key(fields: String!) repeatable on OBJECT | INTERFACE
```

Comment:
This seems wrong - `@link` is fed v2 construct but you are using fed v1 `@key`. What is the result of the composed supergraph?